releases/8.0.0.md

```diff
@@ -148,14 +150,18 @@ JavaScript:
 ### Upgrading an Arches project
 
 1. In pyproject.toml, make these updates:
+    - Update `requires = ["setuptools", "setuptools-scm"]` to `requires = ["setuptools>=77", "setuptools-scm"]`.
     - Update `requires-python` to `">=3.11"`
     - Remove `"Programming Language :: Python :: 3.10"`
     - Add `"Programming Language :: Python :: 3.13"`
     - Remove `"Framework :: Django :: 4.2"`
     - Add `"Framework :: Django :: 5.2"`
+    - Remove any classifier beginning with "License :: "
 
 1. Remove "3.10" from the `python-version` matrix in `.github/workflows/main.yml`.
 
+1. In `MANIFEST.in`, remove "graft *". Add exclusions for local settings files, e.g. `exclude <my_project>/settings_local.py`.
```

Comment:
would it be worth moving this into `updateproject`?

Reply:
/cc @chiatt, I remember before you weighed in against auto-fixing certain files. What about packaging manifests?

Reply:
fair, resolving!